LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -34,45 +28,81 @@ public async Task<ActionResult<RunViewModel>> GetRun([FromRoute] Guid id)
             return NotFound();
         }
 
-        // This is needed because of what we think is a bug with serialisation. The "$type" field
-        // doesn't show up in the response if we simply return Ok(RunViewModel.MapFrom(run)).
-        // And we want that field present for consumers to better discriminate what they're getting.
-        return Content(
-            JsonSerializer.Serialize(RunViewModel.MapFrom(run), options.Value.JsonSerializerOptions),
-            MediaTypeNames.Application.Json
-        );
+        return Ok(RunViewModel.MapFrom(run));
     }
 
     [Authorize]
-    [HttpPost("runs/create")]
-    [SwaggerOperation("Creates a new Run.", OperationId = "createRun")]
+    [HttpPost("/category/{id:long}/runs/create")]
+    [SwaggerOperation("Creates a new Run for a Category with ID `id`. This request is restricted to confirmed Users and Administrators.", OperationId = "createRun")]
     [SwaggerResponse(201)]
-    [SwaggerResponse(401)]
-    [SwaggerResponse(403)]
+    [SwaggerResponse(401, "The client is not logged in.", typeof(ProblemDetails))]
+    [SwaggerResponse(400, Type = typeof(ValidationProblemDetails))]
+    [SwaggerResponse(403, "The requesting User is unauthorized to create Runs.", typeof(ProblemDetails))]
+    [SwaggerResponse(404, "The Category with ID `id` could not be found, or has been deleted. Read `title` for more information.", typeof(ProblemDetails))]
     [SwaggerResponse(422, Type = typeof(ValidationProblemDetails))]
```

Comment:
This should actually be of type ProblemDetails because this only gets thrown when the run type is wrong.

Reply:
I'll switch for now, but I'll add a todo. I believe the real fix is to make sure we do return a ValidationProblemDetails here. We should really standardise the fact that 422s (and 400s) return the same type, though. Other actions 422 with a ValidationProblemDetails. We'll only confuse clients.

Reply:
You're right. For now just make it match what's actually returned though.